buildlib/pr/ucx-py/ucx-py-test.yml

```diff
@@ -0,0 +1,90 @@
+parameters:
+  name: ucx-py test
+  demands: []
+
+jobs:
+  - job: ${{ parameters.name }}
+
+    pool:
+      name: MLNX
+      demands: ${{ parameters.demands }}
+
+    steps:
+      - checkout: self
+        fetchDepth: 100
+        clean: true
+        displayName: Checkout
+      - bash: |
+          set -x
+          source buildlib/az-helpers.sh
+          az_init_modules
+          try_load_cuda_env
+          set -eE
```

Comment:
can we `set -xeE` in line 18?

Reply:
I don't know, we can try, but this was copied from jucx that does it in that order: https://github.com/openucx/ucx/blob/e0bb58f1e96dc9d6a2ed8147dc5fe3d30c51921c/buildlib/jucx/jucx-test.yml#L45-L57

Reply:
@petro-rudenko WDYT?